dCommon/Amf3.h

```diff
@@ -257,10 +258,10 @@ class AMFArrayValue : public AMFBaseValue {
 	 *
 	 * @param key The key to remove from the associative portion
 	 */
-	void Remove(const std::string& key, const bool deleteValue = true) {
+	void Remove(const std::string& key) {
 		const AMFAssociative::const_iterator it = m_Associative.find(key);
 		if (it != m_Associative.cend()) {
-			if (deleteValue) m_Associative.erase(it);
+			m_Associative.erase(it);
```

Comment:
the deleteValue parameter was useless due to the conversion to strictly unique_ptrs